### PR TITLE
Initialize the blog app

### PR DESCRIPTION
**Pull request by l1storez**
Created an application in my project

### CODE REVIEW

**Comment by l1storez** [comment]
I think `blog/migrations` need to add to the ignore

**Comment by l1storez** [comment]
I see my colleague has no docstring in 0001_initial.py 'coz file added to pydocstyle ignore. Do I need to add 0001_initial.py to ignore?

**Comment by webknjaz** [comment]
> I think `blog/migrations` need to add to the pylint ignore 'coz I don't influence the file name

Yes. Anything auto-generated and not managed by you may be excluded particularly or completely from linting.

**Review by webknjaz — CHANGES_REQUESTED** [review]
Don't add useless docstrings. It's unacceptable to ignore the linters.

File: djangogirls/blog/models.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+"""docstring."""
+from django.db import models
+from django.utils import timezone
+
+
+class Post(models.Model):
+    """docstring."""
+
+    author = (
+             'models.ForeignKey'
```
Comment (webknjaz) [inline djangogirls/blog/models.py:10]: Why did you turn this into text?

**Comment by l1storez** [comment]
> Don't add useless docstrings. It's unacceptable to ignore the linters.

I wrote in the first commentary "I will change the content of the docstrings"

File: djangogirls/blog/models.py (hunk 1 above)
Comment (l1storez) [reply djangogirls/blog/models.py:10]: It's my accidental mistake

**Comment by l1storez** [comment]
Change the content of the docstrings and add 0001_initial.py to pre-commit ignore

File: djangogirls/blog/admin.py [hunk 2]
```diff
@@ -0,0 +1 @@
+"""Configuration for blog admin."""
```
Comment (webknjaz) [inline djangogirls/blog/admin.py:1]: I don't see any code adding the model to the admin panel. Where is it?

**Review by webknjaz — COMMENTED** [review]
It seems like this PR mixes up two separate changes:
1) Initializing the blog app
2) Adding a model to that app

Could you separate them?

File: djangogirls/blog/admin.py (hunk 2 above)
Comment (l1storez) [reply djangogirls/blog/admin.py:1]: In the next part of the tutorial.

**Comment by l1storez** [comment]
I created an app then create a blog post model. Split this PR into two PR's? Why?

**Comment by webknjaz** [comment]
> Creating an app is one step

And creating a model is the second step

**Comment by webknjaz** [comment]
> Add 0001_initial.py to pre-commit ignore

This commit message is lying: it doesn't ignore a specific file but the whole folder and it also adds the said file to the repository which is not mentioned in the commit message at all.

**Comment by l1storez** [comment]
> > Add 0001_initial.py to pre-commit ignore
> 
> This commit message is lying: it doesn't ignore a specific file but the whole folder and it also adds the said file to the repository which is not mentioned in the commit message at all.

it's my mistake, I will change this

**Comment by l1storez** [comment]
> And creating a model is the second step

I mean one step is one command(`python manage.py startapp blog`) and add `blog` to INSTALLED_APPS. It's too small changes to separate PR for me

**Comment by webknjaz** [comment]
> I mean one step is one command(`python manage.py startapp blog`) and add `blog` to INSTALLED_APPS.

This looks like one step. Did it also pre-populate the model and the migration with this command? If not, the model creation doesn't belong with it.

**Comment by l1storez** [comment]
I don't understand the scheme for creating PR according to the tutorial, 'coz at first, there was 1 part of the tutorial- 1 PR. Now combine 2 parts in 1 or separate into 1 into parts. I'm confused

**Comment by l1storez** [comment]
Leave the PR as it is or divides?

**Comment by l1storez** [comment]
WHAT?

**Review by webknjaz — COMMENTED** [review]
This PR does not add a model as the title/description states. Fix it.

**Review by webknjaz — CHANGES_REQUESTED** [review]
This PR does not add a model as the title/description states. Fix it.

**Comment by l1storez** [comment]
okay

**Comment by l1storez** [comment]
Corrected title/description

**Comment by webknjaz** [comment]
> Corrected title/description

It would be better if the description didn't contain just a title but contained an actual description.

**Comment by l1storez** [comment]
Okay

**Review by webknjaz — APPROVED** [review]

**Pull request merged**